modules/react/text/stories/Text.stories.mdx

```diff
@@ -1,132 +1,54 @@
 
-# Canvas Kit React Text
+# Canvas Kit Text
 
-`Text` is a group of text components that provide a common, ergonomic API to render text with
-simplified usage of Canvas Kit type token.
+`Text` provides an ergonomic API for rendering text. It includes built-in support for Canvas type
+tokens.
 
 ## Installation
 
 ```sh
 yarn add @workday/canvas-kit-react
 ```
 
-## Text
+## Usage
 
-### Usage
+### Basic Example
 
-`Text` is the base component that is used to render text in the UI. It is built off of our `Box`
-component which allows the use of styles props such as `display`, `margin`, `padding` as well as the
-most common type styles such as `fontWeight`, `fontSize` amongst others. This component is an
-extension of our [type tokens](https://canvas.workdaydesign.com/tokens/type/#type-styles),
-therefore, props like `typeLevel` and `variant` all exists.
+`Text` is built on top of [`Box`](/components/layout/box/) and supports a multitude of
+[style props](/getting-started/for-developers/resources/style-props/) including spacing props such
+as `margin` and `padding`, as well as text styling props such as `fontSize` and `fontWeight`.
 
-By default `Text` component is a `span` element, but it's possible to override the element by
-passing `as` prop.
+`Text` renders a `<span>` element by default. You may override the rendered element using the `as`
+prop.
 
-```tsx
-<Text as="p" fontSize={14} fontWeight="medium">
-  Text here...
-</Text>
-```
-
-<ExampleCodeBlock code={BasicText} />
-
-### Text Props
-
-As previously mentioned, `Text` is built on top of `Box` and has access to all its props. In
-addition `Text` has its own props:
-
-<ArgsTable of={TextComponent} />
-
-## LabelText
-
-`LabelText` is a `label` element build based on the top of `Text` component with added ability to
-add a cursor or disabled style.
-
-### Usage
+<ExampleCodeBlock code={Basic} />
 
-```tsx
-<LabelText cursor="pointer" disabled={true}>
-  Text here...
-</LabelText>
-```
-
-<ExampleCodeBlock code={Label} />
-
-### LabelText Props
+### Type Level and Variant
 
-In addition to `Text` component props, `LabelText` has the following props:
+`Text` includes built-in support for [Canvas type tokens](/tokens/type/) via the `typeLevel` and
+`variant` props.
 
-<ArgsTable of={LabelComponent} />
+`typeLevel` hooks into the [Canvas type hierarchy](/tokens/type/#type-styles).
 
-## Type Level Components
+<ExampleCodeBlock code={TypeLevel} />
 
-Type Level Components, `Subtext`, `BodyText`, `Heading` and `Title`, apply type token styles to text
-element. Instead of passing type level token you can use one of Type Level Components, like in
-example below:
+`variant` allows you to layer [additional styling](/tokens/type/#variants) to aid users'
+understanding of the text being rendered.
 
-```tsx
-// v7
-<p css={{...type.levels.subtext.small}}>Small Subtext Text</p>
-
-// v8
-<Subtext size="small">Small Subtext Text</Subtext>
-```
+<ExampleCodeBlock code={Variant} />
```

Comment:
I created additional examples for `typeLevel` and `variant`.

Reply:
Thank you!